pipelines/production/deployment.yaml

```diff
@@ -18,6 +19,17 @@ jobs:
       environment: ${{ parameters.environment }}
       cipEnvironment: ${{ parameters.cipEnvironment }}
 
+  - template: ../data-pipeline/deployment.yaml
+    parameters:
+      dependsOn: 'CoreProvision'
+      subscription: ${{ parameters.subscription }}
+      environmentPrefix: ${{ parameters.environmentPrefix }}
+      environment: ${{ parameters.environment }}
+      cipEnvironment: ${{ parameters.cipEnvironment }}
+      importImage: true
+      sourceEnvironmentPrefix: 's198t01'
+      buildNumber: ${{ parameters.pipelineBuildNumber }}
+
   - template: ../platform/deployment.yaml
     parameters:
       dependsOn: 'CoreProvision'
```

Comment:
Either platform needs to be dependent on pipeline or web dependent on both platform & pipeline.